kqueen/engines/gce.py

```diff
@@ -429,7 +429,7 @@ def cluster_get(self):
             'name': self.cluster_id,
             'id': self.cluster.id,
             'state': state,
-            'metadata': {}
+            'metadata': {'status_message': response['statusMessage']} if 'statusMessage' in response else {}
```

Comment:
`response['statusMessage']} if 'statusMessage' in response else {}`
->
`response.get('statusMessage', {})`

Reply:
No, I need empty dictionary or dictionary with the 'status_message' key

Reply:
Oh, didn't notice, okay